tests/neo-vm.Tests/Tests/OpCodes/Splice/CAT.json

```diff
@@ -215,6 +215,112 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "CAT 0x01 with empty ByteArray",
```

Comment:
Afraid of something special here? :thinking:

Reply:
Let's add a "description" json field. It may help explaining the reasoning behind the tests (since json doesnt have comments anyway...)